src/storage/storage.py

```diff
@@ -119,6 +124,36 @@
 
 app = Flask(__name__)
 
+JAEGER_AGENT = os.environ.get("F7T_JAEGER_AGENT", "")
+if JAEGER_AGENT != "":
+    config = Config(
+        config={'sampler': {'type': 'const', 'param': 1 },
+            'local_agent': {'reporting_host': JAEGER_AGENT, 'reporting_port': 5775 },
+            'logging': True,
+            'reporter_batch_size': 1},
+            service_name = "storage")
+    jaeger_tracer = config.initialize_tracer()
+    tracing = FlaskTracing(jaeger_tracer, True, app)
+else:
+    jaeger_tracer = None
+    tracing = None
+
+
+def get_tracing_headers(req):
```

Comment:
Just a comment that this function maybe can be on the common functions file since I see it is used in almost all microservices